sites/models.py

```diff
@@ -278,4 +278,4 @@ def save(self, *args, **kwargs):
         return super().save(*args, **kwargs)
 
     class Meta:
-        verbose_name_plural = 'site categories'
+        verbose_name_plural = 'site regions'
```

Comment:
I believe this actually isn't strictly necessary since Django automatically pluralizes model names by appending an `s`. It was necessary when the model name was `category` to prevent it from pluralizing as `categorys`. However it's also harmless and explicit, so we can just keep it